### PR TITLE
NIFI-7098 - Added count file count information in ListSFTP processor

### PR DESCRIPTION
**Pull request by mail2atulm**
Thank you for submitting a contribution to Apache NiFi.

Please provide a short description of the PR here:

#### Description of PR
Added file count information along with batch id in flow file for ListSFTP processor

_Enables X functionality; fixes bug NIFI-7098

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [* ] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [* ] Does your PR title start with **NIFI-XXXX** where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [* ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?

- [x] Is your initial contribution a single, squashed commit? _Additional commits in response to PR reviewer feedback should be made on this branch and pushed to allow change tracking. Do not `squash` or use `--force` when pushing to allow for clean monitoring of changes._

### For code changes:
- [ *] Have you ensured that the full suite of tests is executed via `mvn -Pcontrib-check clean install` at the root `nifi` folder?
- [x] Have you written or updated unit tests to verify your changes?
- [ *] Have you verified that the full build is successful on both JDK 8 and JDK 11?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the `LICENSE` file, including the main `LICENSE` file under `nifi-assembly`?
- [ ] If applicable, have you updated the `NOTICE` file, including the main `NOTICE` file found under `nifi-assembly`?
- [ ] If adding new Properties, have you added `.displayName` in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by pvillard31** [comment]
Hi @mail2atulm - thanks for the contribution. Is there a JIRA associated to this pull request? If not, can you create one to explain the improvement and use case? https://issues.apache.org/jira/browse/NIFI

**Comment by Yogeshmsharma-architect** [comment]
@pvillard31 yes there is a JIRA. Ticket number: https://issues.apache.org/jira/browse/NIFI-7098

File: nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListFile.java [hunk 1]
```diff
@@ -280,6 +280,8 @@
     public static final String FILE_SIZE_ATTRIBUTE = "file.size";
     public static final String FILE_OWNER_ATTRIBUTE = "file.owner";
     public static final String FILE_GROUP_ATTRIBUTE = "file.group";
+    public static final String FILE_LIST_BATCH_COUNT = "file.list.batch.count";
```
Comment (Yogeshmsharma-architect) [inline nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListFile.java]: I think we also need to have a description of these attributes. 
        @WritesAttribute(attribute=ListFile.FILE_LIST_BATCH_COUNT , description="Batch Count...."),

File: nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListFile.java [hunk 2]
```diff
@@ -280,6 +280,8 @@
...
+    public static final String FILE_LIST_BATCH_ID = "file.list.batch.id";
```
Comment (Yogeshmsharma-architect) [inline nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListFile.java]: write attribute

File: nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListFile.java (hunk 1 above)
Comment (Yogeshmsharma-architect) [inline nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListFile.java]: Small description in CapabilityDescription will be also needed

**Pull request closed without merging**